### ISSUE TITLE
Do not require fresh latency calibration on refresh

### ISSUE THREAD

**Issue by taymonbeal** [gwillen/solstice-audio-test#27]
There will likely be circumstances wherein a user needs to refresh, and it would be good if they could do so and then quickly resume participating.

**Comment by gwillen** [comment]
My observations: since we set the latency hint to "playback" (or whatever we changed it to), the latency is generally pretty stable between runs (varying by 5ms or less). I strongly agree that we should not force the user to re-calibrate latency (or volume, now) on every run.

Once we save it, we should give them the option to recalibrate it. And we might want to suggest it, if we see their audio devices change. (But other than Bluetooth, which we discourage, I suspect that most audio devices on the same system will have at least vaguely similar latency, so they can probably get by without, and we should't force it.)
